### DISCUSSION TITLE
Can Sun Ray 2 thin client be a LTSP client?

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#322] [General]
Hello!
How are you?

Please, is possible Sun Ray 2 thin client works as ltsp client?
Or only with Sun Ray server?

Thank You!

**Comment by alkisg** [comment]
Hi Douglas,

if it can run Debian, then it can run LTSP too. So check if it can boot Debian locally.

If it cannot run Debian, then check if you can use it for remote desktop (RDP) without LTSP.

**Reply by doguibnu** [reply]
Hello @alkisg 
Reading about that device seems be very old from 2005/2006 and seeing yours specs, I do not know if it can be ltsp client. Still reading it works with sun ray server. There is a Ubuntu documentation from 2005 how to install sun ray service in linux. I think have must work to try the device works well . Thanks
